### ISSUE TITLE
Unexpected Status Code 404 when trying to run npm install

### ISSUE THREAD

**Issue by anotheredward** [alexanderGugel/ied#2]
After running `ied install` I get the following error message on OSX:

```
[==============================] 100.00%/Users/ed/.nvm/versions/node/v4.2.1/lib/node_modules/ied/lib/install_cmd.js:63
        if (err) throw err
                 ^

Error: Unexpected status code: 404
    at ClientRequest.<anonymous> (/Users/ed/.nvm/versions/node/v4.2.1/lib/node_modules/ied/lib/resolve.js:37:17)
    at ClientRequest.g (events.js:260:16)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:415:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:88:23)
    at Socket.socketOnData (_http_client.js:305:20)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
```

**Comment by alexanderGugel** [comment]
Are you using the public npm registry? What package were you trying to install? Does it actually exist?

The way we currently handle errors is to just blow up the entire thing, which is arguably not the best idea.

Thanks for reporting this.

**Comment by anotheredward** [comment]
Ok, so I went and triaged the problem, looked like it was one of our private modules:
https://www.npmjs.com/private-modules
eg:
```json
https://www.npmjs.com/private-modules

**Comment by alexanderGugel** [comment]
Yup. Scoped modules currently don't work. Good point! Will be changed!

**Comment by alexanderGugel** [comment]
Fixed error messages in v0.0.3. Scoped modules are coming soon, but require some more work concerning authentication and stuff like that.
